components/Chat/Chat.tsx

```diff
@@ -606,36 +606,40 @@ export function ChatPreview() {
 
 const useStyles = () => {
   const colorScheme = useColorScheme();
-  return StyleSheet.create({
-    chatContainer: {
-      flex: 1,
-      justifyContent: "flex-end",
-      backgroundColor: backgroundColor(colorScheme),
-    },
-    chatContent: {
-      backgroundColor: backgroundColor(colorScheme),
-      flex: 1,
-    },
-    chatPreviewContent: {
-      backgroundColor: backgroundColor(colorScheme),
-      flex: 1,
-      paddingBottom: 0,
-    },
-    chat: {
-      backgroundColor: backgroundColor(colorScheme),
-    },
-    inputBottomFiller: {
-      position: "absolute",
-      width: "100%",
-      bottom: 0,
-      backgroundColor: backgroundColor(colorScheme),
-      zIndex: 0,
-    },
-    inChatRecommendations: {
-      borderBottomWidth: 0.5,
-      borderBottomColor: itemSeparatorColor(colorScheme),
-      marginHorizontal: 20,
-      marginBottom: 10,
-    },
-  });
+  return useMemo(
```

Comment:
I wonder if that does anything? The useStyles hook only already depends on the useColorScheme hook so is only executed when this one changes? But I might be wrong and it executes every time the Chat component rerenders?

Reply:
@nmalzieu it was updating on every rerender!
So it's likely best to use useMemo in all useStyles usage